system/Session/Session.php

```diff
@@ -56,86 +64,151 @@ class Session implements SessionInterface
+	 * @var string 
+	 */
 	protected $sessionDriverName;
 
+	/**
+	 * The session cookie name, must contain only [0-9a-z_-] characters.
+	 * @var string 
+	 */
 	protected $sessionCookieName = 'ci_session';
 
+	/**
+	 * The number of SECONDS you want the session to last.
+	 * Setting it to 0 (zero) means expire when the browser is closed.
+	 * @var int 
+	 */
 	protected $sessionExpiration = 7200;
 
+	/**
+	 * The location to save sessions to, driver dependent..
+	 *
+	 * For the 'files' driver, it's a path to a writable directory.
+	 * WARNING: Only absolute paths are supported!
+	 *
+	 * For the 'database' driver, it's a table name.
+	 * 
+	 * TODO: address memcache & redis needs
+	 *
+	 * IMPORTANT: You are REQUIRED to set a valid save path!
+	 * 
+	 * @var string
+	 */
 	protected $sessionSavePath = null;
 
+	/**
+	 * Whether to match the user's IP address when reading the session data.
+	 *
+	 * WARNING: If you're using the database driver, don't forget to update
+	 * your session table's PRIMARY KEY when changing this setting.
+	 * 
+	 * @var bool 
+	 */
 	protected $sessionMatchIP = false;
 
+	/**
+	 * How many seconds between CI regenerating the session ID.
+	 * @var int 
+	 */
 	protected $sessionTimeToUpdate = 300;
 
+	/**
+	 * Whether to destroy session data associated with the old session ID
+	 * when auto-regenerating the session ID. When set to FALSE, the data
+	 * will be later deleted by the garbage collector.
+	 * @var bool 
+	 */
 	protected $sessionRegenerateDestroy = false;
 
-	protected $cookiePrefix = '';
-
+	/**
+	 * The domain name to use for cookies.
+	 * Set to .your-domain.com for site-wide cookies.
+	 * @var string
+	 */
 	protected $cookieDomain = '';
 
+	/**
+	 * Path used for storing cookies.
+	 * Typically will be a forward slash.
+	 * @var string 
+	 */
 	protected $cookiePath = '/';
 
+	/**
+	 * Cookie will only be set if a secure HTTPS connection exists.
+	 * @var bool 
+	 */
 	protected $cookieSecure = false;
 
 	/**
+	 * Logger instance to record error messages and awarnings.
 	 * @var \PSR\Log\LoggerInterface
 	 */
 	protected $logger;
 
 	//--------------------------------------------------------------------
 
-	public function __construct(\SessionHandlerInterface $driver, $config)
+	/**
+	 * Constructor.
+	 * 
+	 * Extract configuration settings and save them here.
+	 * 
+	 * @param \SessionHandlerInterface $driver
+	 * @param \Config\App $config
+	 */
+	public function __construct(\SessionHandlerInterface $driver, \Config\App $config)
 	{
 		$this->driver = $driver;
 
-		$this->sessionDriverName        = $config->sessionDriver;
-		$this->sessionCookieName        = $config->sessionCookieName;
-		$this->sessionExpiration        = $config->sessionExpiration;
-		$this->sessionSavePath          = $config->sessionSavePath;
-		$this->sessionMatchIP           = $config->sessionMatchIP;
-		$this->sessionTimeToUpdate      = $config->sessionTimeToUpdate;
+		$this->sessionDriverName = $config->sessionDriver;
```

Comment:
I think your setup in Netbeans might be automatically reformatting. Our styleguide doesn't really specify one way or the other, but is there another reason you'd remove the alignment of these columns?